app/views/new_administrateur/procedures/show.html.haml

```diff
@@ -42,25 +55,37 @@
           %p.card-admin-status-todo À faire
       %div
         %p.card-admin-title Service
-        %p.card-admin-subtitle Choix du service de la démarche
+        %p.card-admin-subtitle
+          - if @procedure.service_id.present?
+            = @procedure.service.nom
+          - else
+            Choix du service administratif
       .card-admin-action
-        = button_to 'Modifier', new_admin_service_path, params: { procedure_id: @procedure.id }, method: :get, class: 'card-admin-action-btn'
+        - if @procedure.service_id.present?
+          = link_to 'Modifier', edit_admin_service_path(@procedure.service, procedure_id: @procedure.id), class: 'button'
+        - else
+          = link_to 'Remplir', new_admin_service_path(procedure_id: @procedure.id), class: 'button'
 
     .card-admin
       %div
         %span.icon.accept
         %p.card-admin-status-accept Validé
       %div
         %p.card-admin-title
-          Administrateurs
           %span.badge.baseline
             = @procedure.administrateurs.count
```

Comment:
Ici ce code génère du HTML avec une espace dans le span : `<span class="badge baseline">2 </span>`.

L'espace en rab rend le badge décalé à l'affichage :

<img width="239" alt="Capture d’écran 2020-07-20 à 14 32 16" src="https://user-images.githubusercontent.com/179923/87937931-f9e9ac00-ca95-11ea-9ba0-54be32e37df5.png">

Y'a [plusieurs manières](https://haml.info/docs/yardoc/file.REFERENCE.html#whitespace-removal--and-) de supprimer les espaces intérieurs avec du haml. Mais ici le plus simple serait sans doute de mettre la balise et le texte sur la même ligne :

```suggestion
          %span.badge.baseline= @procedure.administrateurs.count
```

Et ça corrige l'alignement :

<img width="234" alt="Capture d’écran 2020-07-20 à 14 36 17" src="https://user-images.githubusercontent.com/179923/87938166-6a90c880-ca96-11ea-94d0-a365c76a91e5.png">